woocommerce/woocommerce
# Create shipping fee field block and initial shipping section

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Closes https://github.com/woocommerce/woocommerce/issues/37406



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Got to `/wp-admin/tools.php?page=woocommerce-admin-test-helper`
2. Under Features tab make sure to enable `block-editor-feature-enabled`
3. Then visit `/wp-admin/admin.php?page=wc-admin&path=/add-product`
4. Under the Shipping tab in the Fees & dimensions section the shipping fee fields should be shown

https://user-images.githubusercontent.com/13334210/230967098-c8e4d335-5270-4033-b4f9-439ce05c3ea9.mov
